### ISSUE TITLE
Foxx configuration is lost when foxx service is replaced

### ISSUE THREAD

**Issue by hwde** [arangodb/arangodb#7011]
## My Environment

* __ArangoDB Version__:        3.3.17
* __Storage Engine__:             RocksDB
* __Deployment Mode__:       Single Server
* __Deployment Strategy__:   ArangoDB Starter
* __Configuration__:               -
* __Infrastructure__:               own
* __Operating System__:        Debian 9
* __Used Package__:              Debian

## Component, Query & Data

__Affected feature__: Foxx

## Steps to reproduce

1. Install a Foxx Service which has a configuration option, i.e: from manifest:

   ```
   "configuration": {
        "api_secret": {
            "description": "Cloud Server Secret",
            "default": "",
            "type": "password"
        }
   },
   ```

2. visit service configuration, type something into the "Cloud Server Secret" and save it.

3. Replace the foxx service via the browser (upload .zip), ensure that "Keep configuration and dependency files?:" is checked! ("run setup" makes no difference here).

4. Visit the service configuration, "Cloud Server Secret" is empty.

__Problem__:

Foxx service configuration values should be be preserved when "Keep configuration and dependency files?" is checked.

**Comment by pluma** [comment]
I think this is a UI issue. The checkbox seems to do the opposite of what it indicates.

**Comment by maxkernbach** [comment]
Hi @hwde,

I am closing this issue as fixed.
The misleading description when replacing a Foxx application is corrected in 3.4.6.

Please find the latest release in the [download section](https://www.arangodb.com/download-major/).
